json-schema-form.d.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Shorthand to lookup for keys with `x-jsf-*` preffix.
+ */
+export function pickXKey(node: Object, key: 'presentation' | 'error-message'): Object | undefined;
+
+type JSConfig = {
+  initialValues?: Record<string, unknown>;
+  strictInputType?: boolean;
+  customProperties?: {
+    description: Function | string;
```

Comment:
question: Can we type this Function better? It must return a string. The Fn params are `(description: string, field: object)`. The `field` is complex to type, we could use the `$TsFixMe`
([Source code](https://github.com/remoteoss/json-schema-form/blob/main/src/internals/helpers.js#L42))

Reply:
Done!